### DISCUSSION TITLE
视频下载功能 希望后续支持8K源的下载

### DISCUSSION THREAD

**Discussion by Lichtmann** [the1812/Bilibili-Evolved#3490] [功能建议 / Ideas]
chorme浏览器端能打开8K视频源了，期待后续下载功能！！！

**Comment by the1812** [comment]
本来就支持啊
<img width="400" src="https://user-images.githubusercontent.com/26504152/178380607-9d2aeb0b-ca1a-4dfe-9d4d-30b1c76a7114.png">
